index.html

```diff
@@ -49,5 +48,176 @@ <h2>I'm a Software Developer</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <div id="letsConectLik">LET'S CONNECT</div>
+                <div class="socialLinks_AboutMe">
+                    <ul class="horizontal-list">
+                        <li><a href="javascript:void(0);"><img id="gatoIconBot" src="images/Iconos/Gato.png" alt="iconGato"></a></li>
+                        <li><a href="javascript:void(0);"><img id="gatoIconBot" src="images/Iconos/Linkedin.png" alt="iconLinkedin"></a></li>
+                        <li><a href="javascript:void(0);"><img id="gatoIconBot" src="images/Iconos/Manito.png" alt="iconManito"></a></li>
+                        <li><a href="javascript:void(0);"><img id="gatoIconBot" src="images/Iconos/Twitter.png" alt="iconTwitter"></a></li>
+                        <li><a href="javascript:void(0);"><img id="gatoIconBot" src="images/Iconos/M.png" alt="iconM"></a></li>
+                    </ul>
+                </div>
+                <div class="workButton">
+                    Get my resume
+                </div>
+                <hr class="aboutMeSeparation">
+            </div>
+        </section>
+        <section>
+            <div class="technologies">
+                <div class="tech">
+                    <div class="openable">
+                        <p>Languages</p>
+                        <img src="images/Techs/icons8-arrow-down-50.png">
+                    </div>
+                    <div class="techflex">
+                        <div class="stackNotice">
+                            <img src="images/Techs/icons8-javascript-48.png">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="stackNotice">
+                            <img src="images/Techs/icons8-html5-48.png">
+                            <p>HTML</p>
+                        </div>
+                        <div class="stackNotice">
+                            <img src="images/Techs/icons8-css3-48.png">
+                            <p>CSS</p>
+                        </div>
+                    </div>
+                </div>
+                <hr class="TechSeparation">
+                <div class="tech">
+                    <div class="openable">
+                        <p>Frameworks</p>
+                        <img src="images/Techs/icons8-arrow-right-50.png">
+                    </div>
+                </div>
+                <hr class="TechSeparation">
+                <div class="tech">
+                    <div class="openable">
+                        <p>Skills</p>
+                        <img src="images/Techs/icons8-arrow-right-50.png">
+                    </div>
+                </div>
+                <hr class="TechSeparation">
+            </div>
```

Comment:
- [ ] Great job 👍🏿  on including the technology categories! However, to align with the design requirements, please make the following updates:

Instead of having the technology categories as separate elements, let's group them within a `<ul>` element, with each category represented as an `<li>` item. This will help maintain a consistent and organized structure, as specified in the design.

Here's an example of how the code structure should look so please implement it with your design:

```html
<ul>
  <li>Languages
    <ul>
      <li>JavaScript</li>
      <li>HTML</li>
      <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks
    <ul>
      <li>React</li>
      <li>Angular</li>
      <li>Vue.js</li>
    </ul>
  </li>
  <li>Skills
    <ul>
      <li>Problem Solving</li>
      <li>Collaboration</li>
      <li>Communication</li>
    </ul>
  </li>
</ul>
```

By implementing the technology categories in this format, you will adhere to the design and maintain a consistent and organized structure.

- [ ] To further align with the design requirements, please make sure of the following:

1. Confirm that the content within each project section is aligned using Flexbox, as per the design requirements. This will help create a flexible and consistent layout.

2. Check that the section has the correct margins (top, bottom, left, and right) defined, as specified in the design. Properly set margins will ensure the appropriate spacing and visual balance.

3. Verify that the section uses the designated font and colors (background and font) defined for the template. Consistency in typography and color scheme contributes to a cohesive design.

- [ ] I noticed some differences between the current design implementation and the provided Figma design. It's crucial to ensure that the final result matches the Figma design accurately. Carefully review the Figma design and compare it with your current implementation.

- [ ] Your design 
![image](https://github.com/alexansaa/Personal_Portafolio/assets/68823331/99750366-f793-4f82-a909-3867fd35c0de)

- [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
![image](https://github.com/alexansaa/Personal_Portafolio/assets/68823331/a6ac741c-ece8-40bf-a0b6-0876ffa1c3fd)